com.unity.render-pipelines.universal/Shaders/ComplexLit.shader

```diff
@@ -134,6 +134,67 @@ Shader "Universal Render Pipeline/Complex Lit"
             #include "Packages/com.unity.render-pipelines.universal/Shaders/LitForwardPass.hlsl"
             ENDHLSL
         }
+
+        Pass
+        {
+            Name "Debug Material"
+            Tags{"LightMode" = "DebugMaterial"}
```

Comment:
The Fallback directive has been fixed and there's a PR for ComplexLit that removes the workarounds. I'm just wondering if we could use the Fallback for LightMode Pass too and redirect all of these shaders to one shader that would have these passes.

Reply:
That sounds possible - I don't know much about the fallbacks, would that need to separate per shader though (since the compiler pragmas may well be different from shader-to-shader)?